package.json

```
@@ -98,8 +98,8 @@
     "@nrwl/tao": "13.4.6",
     "@nrwl/web": "13.4.6",
     "@nrwl/workspace": "13.4.6",
-    "@tailwindcss/aspect-ratio": "0.4.0",
-    "@tailwindcss/line-clamp": "0.3.0",
+    "@tailwindcss/aspect-ratio": "^0.4.0",
```

Typically we stick with fixed versions in the `package.json` file. So please remove the `^` here and below.

```suggestion
    "@tailwindcss/aspect-ratio": "0.4.0",
```